### ISSUE TITLE
[Request] Please transform the RetroBar to a platform independent "LastBar"

### ISSUE THREAD

**Issue by SlitEye** [dremin/RetroBar#403]
Hello together, 

I know, that .NET 6 don't support all Windows-supported APIs on Linux and macOS, but your choice to implement RetroBar on a platform independent Runtime gives the chance to implement a LAST needed platform independent and portable DeskBar. 

Platform independent: With .NET 6 Runtime
Portable: Already implemented by design.

@Developers:
I don't know what .NET 6-API's you're using for the RetroBar-Implementation, but would it be realistic for you to imagine that you could make your RetroBar implementation platform independent with .NET 6? 

The idea, to use RetroBar as plattform independent DeskBar everywhere with the same Design/Theme, Icon-Launchers, SysTray and DeskBar-Config on each Windows- and Linux [and maybe also for macOS] is really nice.

**Comment by xoascf** [comment]
https://github.com/dremin/RetroBar/issues/279#issuecomment-1204526432

That would involve implementation for each system and environment in ManagedShell anyway, but since ManagedShell specializes (for the moment) in Windows, quite a bit of time would have to be spent writing each of those managed functions for each of those environments, but since things change rapidly in those environments with each update, there would be no time to ensure any stability for RetroBar, unless someone volunteered to actively collaborate on the code.

**Comment by SlitEye** [comment]
Of course. This would also be more of a long-term goal. I assume that a cross-platform GUI standard for .NET will come for Windows, Linux and macOS (e.g. MAUI).

**Comment by RokeJulianLockhart** [comment]
This would be superb. Although I love PlasmaShell panels, being able to use RetroBar on Windows 7, 10, 11, and Linux would allow some much needed consistency in my life.

**Comment by ikkqqstudio** [comment]
Any Updates On This?
